go/node/docker_node.go

```diff
@@ -204,25 +200,27 @@ func (d *DockerNode) startEnclave() error {
 }
 
 func (d *DockerNode) startEdgelessDB() error {
-	if !d.cfg.sgxEnabled {
-		// Non-SGX hardware use sqlite database so EdgelessDB is not required.
-		return nil
-	}
-
 	envs := map[string]string{
 		"EDG_EDB_CERT_DNS": d.cfg.nodeName + "-edgelessdb",
 	}
+	devices := map[string]string{}
 
-	devices := map[string]string{
-		"/dev/sgx_enclave":   "/dev/sgx_enclave",
-		"/dev/sgx_provision": "/dev/sgx_provision",
+	if d.cfg.sgxEnabled {
+		devices["/dev/sgx_enclave"] = "/dev/sgx_enclave"
+		devices["/dev/sgx_provision"] = "/dev/sgx_provision"
+	} else {
+		envs["OE_SIMULATION"] = "1"
 	}
 
 	// only set the pccsAddr env var if it's defined
 	if d.cfg.pccsAddr != "" {
 		envs["PCCS_ADDR"] = d.cfg.pccsAddr
 	}
 
+	// todo - do we need this volume?
```

Comment:
@BedrockSquirrel , can you think of any situation where we need the data between different runs of the testnet? 
(Other than upgrades, which we don't test locally currently?)

Reply:
I'm a bit confused, yeah it's just for upgrades. But doesn't this code gets used for the main testnets as well as the local testnets?

Reply:
Worth deleting this then, or want to leave it in case we need to add it back in?

Reply:
yeah. there might be scenarios we don't know